src/mapnik/Mapnik.java

```diff
@@ -18,37 +16,8 @@
 public class Mapnik {
 	private static boolean initialized;
 	private static boolean registered;
-	private static String loadedLibrary;
-	private static List<String> librarySearchNames=new ArrayList<String>();
```

Comment:
I couldn't get mapnik-jni to find the JNI library on my Mac (even with Mapnik 2), so I've replaced the `librarySearchNames` logic by a single call to `loadLibrary` below. Now I just point `-Djava.library.path=...` to the right directory and everything works.
I can take this commit out of the PR if this does not work in other environments.